src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml

```diff
@@ -140,7 +140,8 @@
                                  Background="{DynamicResource {x:Static rwpf:Brushes.BackgroundBrushKey}}" 
                                  Foreground="{DynamicResource {x:Static rwpf:Brushes.UITextKey}}"
                                  ToolTip="{Binding Path=NameTextBoxTooltip}"
-                                 IsVisibleChanged="TextBoxName_IsVisibleChanged"/>
+                                 IsVisibleChanged="TextBoxName_IsVisibleChanged"
+                                 AutomationProperties.Name="{Binding Path=NameTextBoxTooltip}"/>
```

Comment:
I'm not sure that we need a `Name` automation property for the text box. What bug is this?